loris/resolver.py

```diff
@@ -401,36 +404,59 @@ def resolve(self, app, ident, base_uri):
 
-    The configuration SHOULD contain
-     * `templates`, a comma-separated list of template names e.g.
-        templates=`site1,site2`
-     * A subsection named for each template, e.g. `[[site1]]`. This subsection
-       MUST contain a `url`, which is a url pattern for each specified template, e.g.
-       url='http://example.edu/images/%s' or
-       url='http://example.edu/images/%s/master'. It MAY also contain other keys
-       from the SimpleHTTPResolver configuration to provide a per-template
-       override of these options. Overridable keys are `user`, `pw`,
-       `ssl_check`, `cert`, and `key`.
-
-    Note that if a template is listed but has no pattern configured, the
-    resolver will warn but not fail.
-
-    The configuration may also include the following settings, as used by
-    SimpleHTTPResolver:
-     * `default_format`, the format of images (will use content-type of
-        response if not specified).
-     * `head_resolvable` with value True, whether to make HEAD requests
-        to verify object existence (don't set if using Fedora Commons
-        prior to 3.8).  [Currently must be the same for all templates]
-    '''
+    The configuration should contain:
+
+    *   ``templates``, a comma-separated list of template names.  For example:
+
+            templates = 'site1,site2'
+
+    *   A named subsection for each template.  This subsection MUST contain
+        a ``url``, with a URL pattern for each template.  For example:
+
+            [[site1]]
+            url = 'http://example.edu/images/%'
+
+            [[site2]]
+            url = 'https://example.edu/images/%s/master'
+
+        Each subsection MAY also contain other keys from the SimpleHTTPResolver
+        configuration to provide a per-template override of each of these
+        options -- ``user``, ``pw``, ``ssl_check``, ``cert`` and ``key``.
+
+    If a template is listed but has no pattern configured, the resolver
+    will warn but not error.
+
+    If a template has multiple fill-in sections, you can pass a ``delimiter``
+    option to the global config.  When an identifier is received, it will
+    be split on this delimiter to get the different parts.  For example:
+
+        templates = 'site'
+        delimiter = '|'
+
+        [[site]]
+        url = 'http://example.edu/images/%/dir/%s'
+
+    Making a request for identifier ``site:red|yellow`` would resolve to the
+    URL ``http://example.edu/images/red/dir/yellow``.
+
+    The configuration may also include the following settings, as used
+    by SimpleHTTPResolver:
+
+    *   ``default_format``, the format of images (will use the Content-Type
+        of the response if unspecified)
+    *   ``head_resolvable`` with value True, whether to make HEAD requests
+        to validate object existence (don't set if using Fedora Commons
+        prior to 3.8.)  [Currently must be the same for all templates.]
+
+    """
```

Comment:
Along with some examples, big change here is that the `delimiter` option is now documented – previously it was used in code but never mentioned in docs.